core/converter/bytes-converter.mjson

```diff
@@ -1,34 +1,34 @@
 {
     "bytesConverter_decimals": {
-        "prototype": "core/meta/property-blueprint",
+        "prototype": "core/meta/property-descriptor",
         "values": {
             "name": "decimals",
-            "blueprint": {
+            "objectDescriptor": {
                 "@": "root"
             },
             "valueType": "number",
             "helpKey": ""
         }
     },
-    "converter_blueprint_reference": {
-        "prototype": "core/meta/blueprint-reference",
+    "converter_descriptor_reference": {
+        "prototype": "core/meta/object-descriptor-reference",
```

Comment:
Deprecated. Use { "object": "core/converter/converter.mjson" } instead of an instance of an object-descriptor-reference.